### DISCUSSION TITLE
Missed Recovery code and Authentication code

### DISCUSSION THREAD

**Discussion by HindumathyG** [community/community#52488] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am doing password reset but it is asking to enter authentication code that I don't have. Also, I missed the recovery code as well. So I couldn't reset my password. Is there a way to login. via mobile OTP. It's a blocker for me to access the repository.  Anyone please help me out here
<img width="277" alt="Github" src="https://user-images.githubusercontent.com/28006363/231064552-39594fb0-bcfc-4dd8-832f-32983e9999ed.png">

**Comment by airtower-luna** [comment]
I'm afraid the [account recovery documentation is very clear](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials):

> If you cannot use any recovery methods, you have permanently lost access to your account.

You need to have an OTP (or other 2FA method you have set up), or one of your recovery codes.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @HindumathyG ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
